CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## v2.6.0
 
+### Packaging
+
+* MapboxNavigation now requires [MapboxMaps v10.6.0-beta.1](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.6.0-beta.1). ([#3913](https://github.com/mapbox/mapbox-navigation-ios/pull/3913))
+* MapboxCoreNavigation now requires [MapboxNavigationNative v103.0.1](https://github.com/mapbox/mapbox-navigation-native-ios/releases/tag/103.0.1). ([#3913](https://github.com/mapbox/mapbox-navigation-ios/pull/3913))
+
 ### Routing
 
 * Added `Router.finishRouting()` method to finish routing session without dismissing related UI and logic components. ([#3880](https://github.com/mapbox/mapbox-navigation-ios/pull/3880))
```

Comment:
Rename this section to “Location tracking” (which is more general) and add the following changelog entry:

* Fixed an issue where `RouteController` sometimes took too long to detect that the user went off the route after making a turn or taking an off-ramp. ([#3913](https://github.com/mapbox/mapbox-navigation-ios/pull/3913))

Reply:
Thanks for the suggestion, I'll update this part